.github/workflows/build-test.yml

```diff
@@ -36,25 +36,25 @@ jobs:
 
        - name: Build sources
          run: |
-            $GITHUB_WORKSPACE/tools/iceoryx_build_test.sh build-strict build-all out-of-tree build-shared
+            sudo $GITHUB_WORKSPACE/tools/iceoryx_build_test.sh build-strict build-all out-of-tree build-shared test-add-user
 
        - name: Build debian package
          run: |
-            $GITHUB_WORKSPACE/tools/iceoryx_build_test.sh package
+            sudo $GITHUB_WORKSPACE/tools/iceoryx_build_test.sh package
 
-       - name: Run timeless tests
+       - name: Run all tests, excluding timing_tests
```

Comment:
So, timing tests are run separately now and not in the default test set?

Reply:
Yes, the idea is that we collect all tests with timing constraints into one target to execute them separately in an environment with defined hardware specs. Currently it seems that the timing_tests does not produce too much trouble on while running on GitHub CI.